client/src/components/Footer/Footer.css

```diff
@@ -1,54 +1,102 @@
-.footer-component {
-  position: relative;
-  margin-top: 40px;
-}
-
-.footer-component .footer-surface-img {
-  width: 100%;
-  pointer-events: none;
-}
-.icons {
-  height: 30px;
-  width: 30px;
-  margin-left: 20px;
-  margin-bottom: 10px;
-  transition: ease 0.5s;
-}
-
-.icons:hover {
-  color: #1eabd6;
-  transform: translate(0, -1px);
-}
-.last {
-  padding-top: 15px;
-}
-.last p {
-  font-size: 1.2vw;
-}
-.star-background {
-  background-image: url("./img/star.svg");
-  background-attachment: fixed;
-  background-size: cover;
-  background-position: center;
-  background-repeat: no-repeat;
-  height: auto;
-}
-@media (max-width: 767px) {
-  .footer-surface-img {
-    margin-top: 50px;
-  }
-  .last {
-    position: absolute;
-    background-color: rgba(0, 0, 0, 0.5);
-    bottom: 0px;
-    width: 100%;
-  }
-  .last p {
-    font-size: 2vw;
-  }
-}
-@media (max-width: 576px) {
-  .last p {
-    font-size: 4vw;
-  }
-}
+.footer-component {
+      position: relative;
+      margin-top: 40px;
+}
+
+.footer-component .footer-surface-img {
+      width: 100%;
+      pointer-events: none;
+}
+
+.icons {
+      height: 30px;
+      width: 10vh;
+      margin-left: 50px;
+      margin-right: 50px;

```

Comment:
Maybe you can reduce this to 25px? Its not looking good to me though.. What do you think @hariom1625?

Reply:
Okay, Let's take other suggestions too accordingly I will make the change. @tusharjain0022 @ankiiitraj

Reply:
not an expert rather a newbie :octocat:  in this, so over to @tusharjain0022

Reply:
You can try reducing the margins on small screens as suggested by @EmperorYP7, i think that might solve the issue, else everything looks good to me.

Reply:
I think @EmperorYP7 suggested changes for larger screens because for the mobile view its already less.
A bit of confused !!

Reply:
![Screenshot_2020-10-02-13-13-20-324_com android chrome](https://user-images.githubusercontent.com/59157988/94902445-cb387a80-04b5-11eb-9a87-a96f191056c9.jpg)

Note: This view is from my chrome mobile browser in desktop view (currently I'm not having my laptop with me :smile:  )

Reply:
In the desktop view of the chrome mobile browser, the whole of the home page is breaking. Have to look for some other alternative for the whole page.
For now, I am reducing the margin in between the icons as suggested by @EmperorYP7 .